### ISSUE TITLE
Crash during first time start up

### ISSUE THREAD

**Issue by technomage33** [AllTheMods/ATM-1#116]
It keeps crashing during the first time startup saying GC overhead is exceeded does anyone have a workaround for this.

**Comment by FallenEagle1** [comment]
Yes. That means that you have not allocated enough memory for minecraft to run. I would go through this link https://www.reddit.com/r/allthemods/comments/968r4k/how_to_fix_modpack_wont_load_fps/ and make all of the changes that are suggested.
